Doxyfile

```diff
@@ -38,7 +38,7 @@ PROJECT_NAME           = "iRODS"
 # could be handy for archiving the generated documentation or if some version
 # control system is used.
 
-PROJECT_NUMBER         = main
+PROJECT_NUMBER         = 4.3.1
```

Comment:
This should probably still say main? Not sure

Reply:
yes please leave as main

Reply:
Ultimately this only matters when generating documentation.

I can see it going either way, but to keep things simple, let's set it to `main`.

Perhaps this file becomes a .in file and the project number is assigned at build time. Then we don't ever have to touch this file in relation to the version number again.

Reply:
Good catch.

Reply:
Created https://github.com/irods/irods/issues/7395 for the .in file idea. Resolving.